app/src/main/java/com/example/dogbook/main/models/Post.java

```diff
@@ -53,6 +57,30 @@ public void setAuthor(ParseUser user) {
 
     public void setLocation(ParseGeoPoint location) { put(KEY_LOCATION, location);}
 
+    public boolean isLiked() {
+        return isLiked;
+    }
+
+    public void setLiked(boolean liked) {
+        isLiked = liked;
+    }
+
+    public int getNoLikes() {
+        return noLikes;
+    }
+
+    public void setNoLikes(int noLikes) {
+        this.noLikes = noLikes;
+    }
+
+    public int getNoComments() {
+        return noComments;
+    }
+
+    public void setNoComments(int noComments) {
+        this.noComments = noComments;
+    }
```

Comment:
"No" isn't really a great abbreviation -- this is numberOfComments right or CommentCount?  I'd go for one of those instead